docs/source/workflow_inputs.rst

```diff
@@ -1003,7 +1009,7 @@ Each floor/ceiling surface that is not in contact with the ground (Slab) nor adj
   ``Insulation/AssemblyEffectiveRValue``  double    F-ft2-hr/Btu  > 0          Yes                 Assembly R-value [#]_
   ======================================  ========  ============  ===========  ========  ========  ============================
 
-  .. [#] ExteriorAdjacentTo choices are "outside", "attic - vented", "attic - unvented", "basement - conditioned", "basement - unconditioned", "crawlspace - vented", "crawlspace - unvented", "crawlspace - conditioned", "garage", "other housing unit", "other heated space", "other multifamily buffer space", or "other non-freezing space".
+  .. [#] ExteriorAdjacentTo choices are "outside", "attic - vented", "attic - unvented", "basement - conditioned", "basement - unconditioned", "crawlspace - vented", "crawlspace - unvented", "crawlspace - conditioned", "garage", "other housing unit", "other heated space", "other multifamily buffer space", "other non-freezing space", or "manufactured home underbelly".
```

Comment:
I don't know how best to do this, but it seems like we need to give some guidance on what area and R-value to use. Is it the horizontal floor area adjacent to conditioned space or the surface area of the rounded/rectangular belly adjacent to the underbelly? How does that choice influence the assembly R-value that should be used? I guess I'm just reiterating [this comment](https://github.com/hpxmlwg/hpxml/pull/360#issuecomment-1504059177) because I think it should be called out in the OS-HPXML documentation too.

Reply:
We could document for OpenStudio-HPXML that this should include an adjustment and calculate that in OpenStudio-HEScore.